common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +160,25 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    public static String getAudienceFromAuthority( final AzureActiveDirectoryAuthority authority) {
```

Comment:
Let's move the method to `Authority` class itself

Reply:
mTenantId is a filed supported in AzureActiveDirectoryAudience class hence I chose to keep the method in this class a helper method. If its fits better in authority class I can move it, if need be.

Reply:
I don't feel strongly about. You could also consider writing a completely new util class in Kotlin and put this method there. There are many options. Your call.

Reply:
for now I will let this helper method rest in the audience class for the ease of access, we can chose to move it if the need be